src/main/java/lotto/domain/MatchCriteria.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum MatchCriteria {
+    THREE(3, 5000, 0),
+    FOUR(4, 50000, 0),
+    FIVE(5, 1500000, 0),
+    SIX(6, 2000000000, 0),
```

Comment:
숫자 작성 시 언더 스코어를 활용하면 가독성을 높일 수 있어요!
```suggestion
    SIX(6, 2_000_000_000, 0),
```